whylog/config/investigation_plan.py

```diff
@@ -67,10 +74,15 @@ def __init__(self, regex_parameters, line_prefix_content, line_offset, line_sour
         self.line_source = line_source
 
     def __repr__(self):
+        if all(elem is None for elem in self.__dict__.values()):
```

Comment:
`self.__dict__.values()` contains methods too. This will explode once someone adds a method to this class or to a subclass. Too much magic, please rewrite.